lib/need_for_type/states/state.rb

```diff
@@ -5,5 +5,16 @@ def initialize(display_window)
       @display_window = display_window
     end
 
+    def input_worker(number_option)
+      input = @display_window.get_input
+
+      if input == Curses::Key::UP
+        @option = (@option - 1) % number_option
+      elsif input == Curses::Key::DOWN
+        @option = (@option + 1) % number_option
+      elsif input == Curses::Key::ENTER || input == 10
+        yield
+      end
+    end
```

Comment:
Hmm since they will inherit MenuState I do not think we need to state again that they are Menu, just `Start` and `End` will suffice or even `StartState` and `EndState`